src/routes/article.route.js

```diff
@@ -0,0 +1,33 @@
+import { Router } from 'express';
+import dotenv from 'dotenv';
+import ArticleController from '../controllers/articles.controller';
+import authMiddleware from '../middleware/auth.middleware';
+
```

Comment:
Please add validation middlewares to your implementation. Kindly reachout to @timi-codes to see how this was implemented with `joi`

Reply:
Done